benchmarks/ios/jvm-vs-kotlin-native/build.gradle.kts

```diff
@@ -0,0 +1,83 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+}
+
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenLocal()
+    google()
+    mavenCentral()
+    maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+}
+
+kotlin {
+    jvm("desktop")
+    macosX64 {
+        binaries {
+            executable {
+                entryPoint = "main"
+                freeCompilerArgs += listOf(
+                    "-linker-option", "-framework", "-linker-option", "Metal"
+                )
+            }
+        }
+    }
+    macosArm64 {
+        binaries {
+            executable {
+                entryPoint = "main"
+                freeCompilerArgs += listOf(
+                    "-linker-option", "-framework", "-linker-option", "Metal"
+                )
+            }
+        }
+    }
+
+    sourceSets {
+        val commonMain by getting {
+            dependencies {
+                implementation(compose.ui)
+                implementation(compose.foundation)
+                implementation(compose.material)
+                implementation(compose.runtime)
+                @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
+                implementation(compose.components.resources)
+            }
+        }
+
+        val desktopMain by getting {
+            dependencies {
+                implementation(compose.desktop.currentOs)
```

Comment:
I committed `gradlew.bat` in the last commit (please, check).

And discovered a possible bug in 1.4.1 on `gradlew run`
```
e: file:///D:/Development/Work/compose-multiplatform/benchmarks/ios/jvm-vs-kotlin-native/src/commonMain/kotlin/MeasureComposable.kt:22:37 Unresolved reference: skia
e: file:///D:/Development/Work/compose-multiplatform/benchmarks/ios/jvm-vs-kotlin-native/src/commonMain/kotlin/MeasureComposable.kt:26:23 Cannot access class 'org.jetbrains.skia.Canvas'. Check your module classpath for missing or conflicting dependencies
```

It succesfully runs on macOs, or on 1.4.0

Reply:
Will investigate separetely